stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -144,4 +144,14 @@ public final void setAttributesAsText(Object element, String text) {
   protected void onSetAttributesAsText(E element, String text) {
     mSuper.setAttributesAsText(element, text);
   }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void getStyles(Object element, StyleAccumulator accumulator) {
+    onGetStyles((E) element, accumulator);
```

Comment:
is there any reason why we're getting local styles and then super's styles? (instead of the other way around) ... if you look at `getAttributes`, it first grabs the super's attributes and then calls `onGetAttributes` ("local" attributes)